src/ripple/overlay/impl/OverlayImpl.cpp

```diff
@@ -912,6 +915,61 @@ OverlayImpl::crawl()
     return jv;
 }
 
+Json::Value
+OverlayImpl::crawl_server()
+{
+    Json::Value jv;
+
+    bool const humanReadable = false;
+    bool const admin = false;
+    bool const counters = false;
+
+    Json::Value server_info = app_.getOPs().getServerInfo(humanReadable, admin, counters);
```

Comment:
I'm guessing you were trying to counteract the trouble of comprehending params a call like `getServerInfo(false, false, false)`? I'm not particularly sold on this approach here, but it's fine either way to me.

Reply:
There's not really any overhead, and if it changes later you'll also get a more readable diff: 
`bool const counters = `~~false~~` true;` vs. `(false, false, `~~false~~`true);`

Reply:
This is a hard problem. In my own (non-rippled) code, I either use enum classes for the parameters or call the function like `getServerInfo(/*humanreadable*/ false, /*admin*/ false, /*counters*/ false)`. Of course, you're right that any function with three bool switches should be looked at as ripe for refactoring.